testsuite/tests/libratest/smoke_test.rs

```diff
@@ -1160,7 +1188,7 @@ fn test_client_waypoints() {
         .unwrap()
         .to_string();
     client_proxy
-        .remove_validator(&["remove_validator", &peer_id], true)
+        .remove_validator(&["remove_validator", &peer_id, &operator_private_str], true)
```

Comment:
I think this operation should still be done by the association -- or we should chat briefly about the right format for this.